### PR TITLE
Updating discussions module

### PR DESCRIPTION
**Pull request by jeromesteve202**
Original Issue: https://devtopia.esri.com/dc/hub/issues/4242

I have added a `Reaction` class which makes it possible to store the ID and also be able to delete it easily.

Also, all endpoints have been updated to use https instead of http.

Notebook has also been updated.

### CODE REVIEW

File: arcgishub/discussions.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def update(self, body=None, title=None, discussion=None, geometry=None, appInfo=
         if appInfo:
             payload['appInfo'] = appInfo
         
-        url = 'http://hub.arcgis.com/api/discussions/v1/posts/{}'.format(self.id)
+        url = 'http://hubqa.arcgis.com/api/discussions/v1/posts/{}'.format(self.id)
```
Comment (ManushiM) [inline arcgishub/discussions.py]: is there a particular reason we are pointing this to qa instead of prod?

**Review by ManushiM — CHANGES_REQUESTED** [review]
I like the introduction to discussions in the notebook, very comprehensive and easy to follow!

Cells 3, 4, 7 have a whole json response printed before the output is printed. Do we have hidden print commands in those methods initially added for testing? If testing is the only purpose, you may comment them if not remove them.

File: arcgishub/discussions.py (hunk 1 above)
Comment (jeromesteve202) [reply arcgishub/discussions.py]: @ManushiM There were 2 commits in this PR and looks like you viewed the first commit. In the second commit, you will see that I have the prod endpoints in the `Files Changed` tab. 

I have also removed the print statements from the module and updated the notebook accordingly.

Comment (ManushiM) [reply arcgishub/discussions.py]: ah I just noticed. Looks good to merge!

**Pull request merged**